tests/test_commands/test_deps_convert.py

```diff
@@ -7,19 +7,38 @@
 from dephell.config import Config
 
 
-def test_convert_command(temp_path: Path):
-    from_path = str(temp_path / 'pyproject.toml')
-    to_path = temp_path / 'setup.py'
-    shutil.copy(str(Path('tests') / 'requirements' / 'poetry.toml'), from_path)
-    config = Config()
-    config.attach({
-        'from': dict(format='poetry', path=from_path),
-        'to': dict(format='poetry', path=str(to_path)),
-        'project': str(temp_path),
-    })
-    command = DepsConvertCommand(argv=[], config=config)
-    result = command()
-    assert result is True
-    assert to_path.exists()
-    content = to_path.read_text()
-    assert 'The description of the package' in content
+class TestConvertCommand:
```

Comment:
I like class-based tests for class-level fixtures, but there is no reason to do it: command placed in a separated module and this is already namespace for tests. So, what is your point to do it?

Reply:
I'll be honest, I'm not completely sure why I put them in a class.  I must have intended to make the requirements_dir fixture somehow class-level?  Really don't know.  I'll remove it.